pkg/health/defaults/defaults.go

```diff
@@ -25,4 +25,8 @@ const (
 
 	// ServiceL7PathPort is used for probing service redirect path connectivity with L7
 	ServiceL7PathPort = 4243
+
+	// EndpointName is the name used for the health endpoint, which is also
```

Comment:
c&p nit:

```suggestion
	// HealthEPName is the name used for the health endpoint, which is also
```

Reply:
:+1: originally I was worried it would stutter but I guess it'll actually end up referenced as defaults.HealthEPName. I can fix this up.